### PR TITLE
Add swarm task state REMOVE

### LINKED ISSUE #6203: To doc: Add REMOVE and ORPHANED to TaskState
References: 
https://github.com/docker/docker-ce/blob/v18.03.0-ce-rc3/CHANGELOG.md
https://github.com/moby/moby/pull/36146

Possible doc location:
https://docs.docker.com/engine/swarm/how-swarm-mode-works/swarm-task-states/

### PR DESCRIPTION
**Pull request by gbarr01**
Fixes #6203 

See: https://github.com/docker/swarmkit/pull/2485

### CODE REVIEW

**Comment by gbarr01** [comment]
@thaJeztah , this is the only real change. Good? 

> `REMOVE`    | The task is ready for removal after shutdown succeeds. It will not be kept around in task history.

File: engine/swarm/how-swarm-mode-works/swarm-task-states.md [hunk 1]
```diff
@@ -6,23 +6,18 @@ redirect_from:
...
 
-Tasks are execution units that run once to completion. When a task stops,
-it isn't executed again, but a new task may take its place.
+Tasks are execution units that run once to completion. When a task stops, it isn't executed again, but a new task may take its place.
 
-Tasks advance through a number of states until they complete or fail. Tasks
-are initialized in the `NEW` state. The task progresses forward through a
-number of states, and its state doesn't go backward. For example, a task never
-goes from `COMPLETE` to `RUNNING`.
+Tasks advance through a number of states until they complete or fail. Tasks are initialized in the `NEW` state. The task progresses forward through a number of states, and its state doesn't go backward. For example, a task never goes from `COMPLETE` to `RUNNING`.
```
Comment (thaJeztah) [inline engine/swarm/how-swarm-mode-works/swarm-task-states.md]: Why were these paragraphs un-wrapped? Perhaps a setting in your editor?

Comment (gbarr01) [reply engine/swarm/how-swarm-mode-works/swarm-task-states.md]: It was probably me ... I'll fix it. Just curious, how does that affect you?

Comment (thaJeztah) [reply engine/swarm/how-swarm-mode-works/swarm-task-states.md]: It allows more granular commenting when reviewing, and makes the text more readable when viewing in source view. Also see (e.g.) https://github.com/carwin/markdown-styleguide (I think there's some other places that describe common MarkDown standards

Comment (gbarr01) [reply engine/swarm/how-swarm-mode-works/swarm-task-states.md]: Got it. Thanks.

**Pull request merged**